### PR TITLE
msazonov: hw_01: add all task

### PR DESCRIPTION
**Pull request by eternity0078**

### CODE REVIEW

**Comment by a-rudkovskaia** [comment]
It's convenient to use scala worksheets in this chapter, some tasks are implied to be done in scala REPL as it affects the result (e.g. 3d task)

File: src/main/scala/msazonov/hw_01/Task01.scala [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+object Task01{
+    def main(args: Array[String]): Unit = {
+        //1. In the Scala REPL, type 3. followed by the Tab key. What methods can be applied?
+
+        // Множество различных методов, такие как .toString(), .getClass(), .equals(), .compare() и тд
+
+        //2. In the Scala REPL, compute the square root of 3, and then square that value. By how much does the result differ
+        // from 3? (Hint: The res variables are your friend.)
+
+        var res: Double = math.sqrt(3)
```
Comment (kaliningleb25) [inline src/main/scala/msazonov/hw_01/Task01.scala]: Please use val where it's applicable

**Review by kaliningleb25 — CHANGES_REQUESTED** [review]

File: src/main/scala/msazonov/hw_01/Task01.scala (hunk 1 above)
Comment (eternity0078) [reply src/main/scala/msazonov/hw_01/Task01.scala]: Done

**Comment by eternity0078** [comment]
> It's convenient to use scala worksheets in this chapter, some tasks are implied to be done in scala REPL as it affects the result (e.g. 3d task)

Done

**Review by kaliningleb25 — APPROVED** [review]

**Review by a-rudkovskaia — APPROVED** [review]

**Comment by kaliningleb25** [comment]
@eternity0078 please merge

**Pull request merged**